### PR TITLE
ref(crons): Drop last_state_change from db

### PR DESCRIPTION
**Pull request by evanpurkhiser**
No longer needed after GH-66353

### CODE REVIEW

File: src/sentry/migrations/0664_monitor_drop_last_state_change_db.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                ALTER TABLE "sentry_monitorenvironment" DROP COLUMN "last_state_change";
+                """,
+                    reverse_sql="""
+                ALTER TABLE "sentry_monitorenvironment" ADD COLUMN "last_state_change" timestamptz;
```
Comment (wedamija) [inline src/sentry/migrations/0664_monitor_drop_last_state_change_db.py:33]: Add it as nullable here. Mostly for dev but this reverse will fail otherwise

**Review by wedamija — APPROVED** [review]

File: src/sentry/migrations/0664_monitor_drop_last_state_change_db.py (hunk 1 above)
Comment (evanpurkhiser) [reply src/sentry/migrations/0664_monitor_drop_last_state_change_db.py:33]: I think default it is nullable without `NOT NULL` right?

Comment (evanpurkhiser) [reply src/sentry/migrations/0664_monitor_drop_last_state_change_db.py]: Refrence: https://www.postgresql.org/docs/7.3/sql-altertable.html#:~:text=The%20new%20column%20always%20comes,to%20set%20the%20default%20afterwards.

> In the current implementation of ADD COLUMN, default and NOT NULL clauses for the new column are not supported. The new column always comes into being with all values NULL. You can use the SET DEFAULT form of ALTER TABLE to set the default afterwards.

right?

Comment (wedamija) [reply src/sentry/migrations/0664_monitor_drop_last_state_change_db.py]: I always get this backwards, because in Django it's the opposite!

**Pull request merged**